### PR TITLE
[OSCD] Bad Opsec Powershell Artifacts

### PR DESCRIPTION
**Pull request by invrep-de**
[OSCD] Bad Opsec Powershell Artifacts

### CODE REVIEW

**Comment by invrep-de** [comment]
This detects powershell code artifacts left by attackers in community attack tool powershell payloads as part of bad opsec practices leveraging customized variants of Cobaltstrike Powershell Beacon, PoshC2, Powersploit, Empire, etc.

**Comment by yugoslavskiy** [comment]
Hello @invrep-de! The rule looks perfect. Thank you very much for your contribution!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Comment by e6e6e** [comment]
Hello @invrep-de!
This is a good rule, thank you! 
But it is very similar to rule [powershell_malicious_keywords](https://github.com/Neo23x0/sigma/blob/de5444a81e770ec730aa5e3af69781ab222f021a/rules/windows/powershell/powershell_malicious_keywords.yml). I think it's better to change this PR (update the existing rule).

**Comment by invrep-de** [comment]
> Hello @invrep-de!
> This is a good rule, thank you!
> But it is very similar to rule [powershell_malicious_keywords](https://github.com/Neo23x0/sigma/blob/de5444a81e770ec730aa5e3af69781ab222f021a/rules/windows/powershell/powershell_malicious_keywords.yml). I think it's better to change this PR (update the existing rule).

Hello, Thanks for your feedback. I thought about it initially as well. However, after reviewing the details, I determined that the semantics/idea behind these two rules are quite different so from a blue team's perspective, it would be of value to separate the two as the categories of fp associated with the two/tuning can be quite different. Specifically, the original rule by Florian checks for some cmdlets etc that are not related to bad opsec. The rule that I put together specifically focuses on bad opsec artifacts and how malicious threat actors are *adapting* their payloads (the intention is to expand this in the future as well.) For this reason, the recommendation is to add this rule as-is.

**Comment by invrep-de** [comment]
Resubmitting to address as discussed with @yugoslavskiy

File: rules/windows/process_creation/win_powershell_disable_windef_av.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Powershell Used To Disable Windows Defender AV Security Monitoring
+id: a7ee1722-c3c5-aeff-3212-c777e4733217
+status: experimental
+description: Detects attackers attempting to disable Windows Defender using Powershell
+author: ok @securonix invrep-de, oscd.community
```
Comment (e6e6e) [inline rules/windows/process_creation/win_powershell_disable_windef_av.yml:5]: Hi @invrep-de !
Please add `'` around string values.
It is also better to split the second rule into a new Pull Request.

File: rules/windows/process_creation/win_powershell_disable_windef_av.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+tags:
+    - attack.defense_evasion
+    - attack.t1089      # legacy
+    - attack.t1562.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+	Image|endswidth: '\powershell.exe'
```
Comment (e6e6e) [inline rules/windows/process_creation/win_powershell_disable_windef_av.yml:19]: Plese fix this line.

**Pull request closed without merging**